prow/gerrit/adapter/adapter.go

```diff
@@ -168,6 +168,28 @@ func createRefs(reviewHost string, change client.ChangeInfo, cloneURI *url.URL,
 	return refs, nil
 }
 
+func parseReports(account int, messages ...gerrit.ChangeMessageInfo) map[string]reporter.JobReport {
+	latestReports := map[string]reporter.JobReport{}
+	latestTimes := map[string]time.Time{}
+	for _, message := range messages {
+		// If message status report is not from the prow account ignore
+		if message.Author.AccountID != account {
+			continue
+		}
+		tag := message.Tag
```

Comment:
I don't think we use tags, do we? should this be label instead? 
Although i don't think we get label from the API https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#change-message-info
we might need to parse that from the report too.

Reply:
Also keying on the label might make things complicated if we have multiple jobs that use an empty label, since they aren't reported together in this case.
One alternative might be to key by job, gather the latest reports for each job and then see if each job was failing in their corresponding latest reports (can consider making jobs a set instead of a list in the jobreport struct, if we go down this route).

Reply:
The tag is the label. 

If the label is empty we don't report.

Reply:
hmm, let me investigate that...

Reply:
(Neither of those things seem as immediately true as when I started down this path). Will update this when it's ready for a review

Reply:
Done

(updated to find the latest result for each job, thanks for catching this hilariously embarrassing oversight!)